packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -265,7 +265,7 @@ const Dropdown = ({
                     )}
                     <div
                         className={dropdownDisplayClassName()}
-                        tabIndex={isSingleOption() ? '-1' : '0'}
+                        tabIndex={tab_index}
```

Comment:
won't this break all the current tab indices? (unless it's passed as a prop to every instance of the dropdown).

Should we add a default prop tab index based on the previous logic: `isSingleOption() ? '-1' : '0'` ?

Reply:
Yeah, it does. Since all of them were indexed zero, it would work but keeping it would break the visual layout, = all of them are 0, the last element added to the page would take the focus. 

I've added tab_index to financial assessment page.

Reply:
What about all the other dropdowns? The keyboard events are not working for them if we don't pass tabIndex to them as well - but a default tabIndex would make these changes backwards compatible.